numpy/linalg/linalg.py

```diff
@@ -1609,6 +1609,9 @@ def slogdet(a):
     ----------
     a : (..., M, M) array_like
         Input array, has to be a square 2-D array.
+        .. versionadded:: 1.8.0
```

Comment:
It doesn't work (now that I've untangled all the sphinx versions floating around my system ;))
```
Input array, has to be a square 2-D array. .. versionadded:: 1.8.0 The log-determinants of stacks of matrices can be calculated by providing higher-dimensional arrays.
```
Perhaps the place to add this is in the Notes or summary section. In the summary section, maybe something like
```
summary blah, blah.

.. versionadded:: 1.8.0
Will work on stacked arrays.
```